app/models/rubygem.rb

```diff
@@ -34,8 +34,10 @@ class Rubygem < ApplicationRecord
   before_destroy :mark_unresolved
 
   MFA_RECOMMENDED_THRESHOLD = 165_000_000
+  MFA_REQUIRED_THRESHOLD = 180_000_000
 
   scope :mfa_recommended, -> { joins(:gem_download).where("gem_downloads.count > ?", MFA_RECOMMENDED_THRESHOLD) }
+  scope :mfa_required_from_downloads, -> { joins(:gem_download).where("gem_downloads.count > ?", MFA_REQUIRED_THRESHOLD) }
```

Comment:
I'm not sure if I love this name yet 😅. What alternatives did y'all consider? I found it a bit confusing to wrap my mind around initially. 

Here are some ideas, just spitballing (they're not necessarily amazing ideas either lol):
* `mfa_required_per_download_threshold`
* `mfa_required_per_threshold`
* `mfa_required_as_per_download_threshold`
* `mfa_required_as_per_threshold`
* `mfa_required_from_download_threshold`
* `mfa_required_threshold_reached`
* `mfa_required_reached_threshold`
* `mfa_required_threshold_cleared`
* `mfa_required_cleared_threshold`

There's something about "from_downloads" that throws me off, it feels like it's implying something else.

Reply:
I suggest we just change mfa_required (from policy) to something else and then use mfa_required here so the scopes have the same naming scheme